<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dr_result</class>
 <widget class="QWidget" name="dr_result">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(170, 170, 127);</string>
  </property>
  <widget class="QPushButton" name="clear">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>340</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>490</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="strategyTableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>840</width>
     <height>701</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="delStrPushButton">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>200</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Del</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editStrPushButton">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>80</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="barpic">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>630</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>BarChart</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
